oak-commons/pom.xml

```diff
@@ -103,6 +104,11 @@
       <artifactId>oak-shaded-guava</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
+      <version>4.4</version>
```

Comment:
version should be removed, it's in the parent pom

Reply:
addressed in https://github.com/apache/jackrabbit-oak/pull/1679/commits/d44677ee9468e3dc173ab4fd56c7a6a7445aba00